### DISCUSSION TITLE
Update multiple table using form-view [Sep 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3836] [Discord Threads]
> 🙋‍♂️ **shashi**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### That wouldn't be possible at the moment. As a workaround, you can update columns in a table & add lookup columns in other.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1019390473342820453)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
